src/main/scala/io/iohk/ethereum/blockchain/sync/SyncController.scala

```diff
@@ -103,23 +103,26 @@ class SyncController(
 }
 
 object SyncController {
-  def props(peerManager: ActorRef,
-            appStateStorage: AppStateStorage,
+  def props(appStateStorage: AppStateStorage,
             blockchain: Blockchain,
             blockchainStorages: BlockchainStorages,
             syncStateStorage: FastSyncStateStorage,
             ledger: Ledger,
             validators: Validators,
-            peerMessageBus: ActorRef):
-  Props = Props(new SyncController(peerManager, appStateStorage, blockchain, blockchainStorages,
-    syncStateStorage, ledger, validators, peerMessageBus))
+            actors: DependencyActors):
+  Props = Props(new SyncController(appStateStorage, blockchain, blockchainStorages,
+    syncStateStorage, ledger, validators, actors))
+
+  case class DependencyActors(peerManager: ActorRef, peerMessageBus: ActorRef, pendingTransactionsManager: ActorRef)
```

Comment:
too many arguments for method, and I agree that it was to many
maybe we should provide actors to sync controller in different way